include/libm2k/contextbuilder.hpp

```diff
@@ -99,6 +110,7 @@ class LIBM2K_API ContextBuilder {
  * @{
  */
 
+
```

Comment:
nitpick: extra line

Reply:
The line was added in order to be style-consistent.